### PR TITLE
[deckhouse] add bootstrapped extender

### LINKED ISSUE #5770: [module] add dependency based on cluster bootstrap
### Preflight Checklist

- [x] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

Activate module only if cluster was bootstrapped

### Proposed Solution

```yaml
dependency:
  bootstrapped: true
```

or 

```yaml
dependency:
  cluster-bootstrapped: true
```

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by ipaqsa**
## Description
It provides the bootstrapped extender.

## Why do we need it, and what problem does it solve?
Fixes #5770 

## What is the expected result?
Modules with the bootstrapped requirement will not start until the cluster is bootstrapped.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse
type: feature
summary: Add bootstrapped extender.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'e2e/run/aws'...

**Comment by yalosev** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.29 10508094592 10508094592-1-con-1-29 /sys/deckhouse-oss/install:pr9425 ec2-user@3.65.159.130

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'e2e/run/aws'...

File: testing/cloud_layouts/VCD/Standard/resources.tpl.yaml [hunk 1]
```diff
@@ -74,3 +74,18 @@ metadata:
...
+data:
+  .secret-file: dmFsdWUtMg0KDQo=
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dotfile-cm
+  namespace: d8-system
+data:
+  .file: "content"
```
Comment (miklezzzz) [inline testing/cloud_layouts/VCD/Standard/resources.tpl.yaml:91]: didn't get what it is

**Review by miklezzzz — APPROVED** [review]

**Review by yalosev — APPROVED** [review]

**Pull request merged**